corehq/apps/hqcase/case_helper.py

```diff
@@ -1,6 +1,10 @@
+import uuid
 from dataclasses import dataclass
 
 from corehq.apps.users.models import CouchUser
+from corehq.form_processor.models.cases import CommCareCase
+from corehq.apps.hqcase.utils import submit_case_blocks, get_censored_case_data
```

Comment:
nit: `deidentified_data` seems to be more appropriate for the given context over `censored_data`, also consistent with the UI language.